WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -411,14 +411,8 @@ extension WordPressAppDelegate {
         }
 
         let utility = AppRatingUtility.shared
-        utility.register(section: "notifications", significantEventCount: 5)
-        utility.systemWideSignificantEventCountRequiredForPrompt = 10
+        utility.systemWideSignificantEventCountRequiredForPrompt = 20
```

Comment:
This means, that at least 20 actions should be performed for the App Ratings Prompt to appear.